command/server_dev.go

```diff
@@ -0,0 +1,900 @@
+		return fmt.Errorf("failed to create %s policy: %w", policyName, resp.Error())
+	}
+
+	return nil
+}
+
+func qsEnableUserpass(ctx context.Context, init *vault.InitResult, core *vault.Core, options *quickstartOptions) error {
+	r := &logical.Request{
+		Operation:   logical.UpdateOperation,
+		ClientToken: init.RootToken,
+		Path:        "sys/auth/userpass",
+		Data: map[string]interface{}{
+			"type":        "userpass",
+			"path":        "userpass/",
+			"description": "sample userpass method",
+		},
+	}
+	resp, err := core.HandleRequest(ctx, r)
+	if err != nil {
+		return fmt.Errorf("error creating userpass: %w", err)
+	}
+	if resp.IsError() {
+		return fmt.Errorf("failed to create userpass: %w", resp.Error())
+	}
+	r = &logical.Request{
+		Operation:   logical.UpdateOperation,
+		ClientToken: init.RootToken,
+		Path:        fmt.Sprintf("auth/userpass/users/%s", options.authUserpass.username),
+		Data: map[string]interface{}{
+			"password": options.authUserpass.password,
+		},
+	}
+	resp, err = core.HandleRequest(ctx, r)
+	if err != nil {
+		return fmt.Errorf("error creating userpass user: %w", err)
+	}
+	if resp.IsError() {
+		return fmt.Errorf("failed to create userpass user: %w", resp.Error())
+	}
+
+	return nil
+}
+
+func qsEnableIdentity(ctx context.Context, init *vault.InitResult, core *vault.Core, options *quickstartOptions) error {
+	r := &logical.Request{
+		Operation:   logical.CreateOperation,
+		ClientToken: init.RootToken,
+		Path:        "identity/entity",
+		Data: map[string]interface{}{
+			"name": options.identity.name,
+			"metadata": map[string]string{
+				"organization": "ACME Inc.",
+				"team":         "Squad-2",
+			},
+			"policies": options.identity.policies,
+		},
+	}
+	resp, err := core.HandleRequest(ctx, r)
+	if err != nil {
+		return fmt.Errorf("error creating entity: %w", err)
+	}
+	if resp.IsError() {
+		return fmt.Errorf("failed to create entity: %w", resp.Error())
+	}
+	id, ok := resp.Data["id"].(string)
+	if !ok {
+		return fmt.Errorf("failed to read entity id")
+	}
+
+	options.identity.entityID = id
+
+	r = &logical.Request{
+		Operation:   logical.ReadOperation,
+		ClientToken: init.RootToken,
+		Path:        "sys/auth",
+	}
+	resp, err = core.HandleRequest(ctx, r)
+	if err != nil {
+		return fmt.Errorf("error reading mount accessor: %w", err)
+	}
+	if resp.IsError() {
+		return fmt.Errorf("failed to read mount accessor: %w", resp.Error())
+	}
+	up, ok := resp.Data["userpass/"].(map[string]interface{})
+	if !ok {
+		return fmt.Errorf("failed to read mount accessor")
+	}
+	accessor, ok := up["accessor"].(string)
+	if !ok {
+		return fmt.Errorf("failed to read mount accessor")
+	}
+
+	r = &logical.Request{
+		Operation:   logical.CreateOperation,
+		ClientToken: init.RootToken,
+		Path:        "identity/entity-alias",
+		Data: map[string]interface{}{
+			"name":           options.authUserpass.username,
+			"canonical_id":   id,
+			"mount_accessor": accessor,
+			"custom-metadata": map[string]string{
```

Comment:
Should this be `custom_metadata`? Or is this a weird idiosyncrasy of the identity endpoint...